tests/conftest.py

```diff
@@ -292,15 +291,6 @@ def _fixture_dir(name: str) -> Path:
     return _fixture_dir
 
 
-@pytest.fixture
-def tmp_dir() -> Iterator[str]:
-    dir_ = tempfile.mkdtemp(prefix="poetry_")
```

Comment:
we want to remove the poetry_ prefix?

Reply:
Good question. I did not find any place that would rely on that prefix. The pytest's `tmp_path` fixture ensures there are no path collisions by using it's own (per-run) pytest prefix.